package.json

```diff
@@ -77,6 +77,7 @@
         }
       ],
       "@typescript-eslint/no-floating-promises": 0,
+      "@typescript-eslint/no-unsafe-enum-comparison": 0,
```

Comment:
I disable this rule since there is a lot of comparisons with enums `"string" === SomeEnum.Attr` which would have had to be converted into `"string" === SomeEnum.Attr.toString()`.